airbyte-integrations/connectors/destination-bigquery-denormalized/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="warn" packages="org.apache.logging.log4j.core,io.sentry.log4j2">
+  <Appenders>
+    <Sentry name="Sentry"/>
+  </Appenders>
+  <Loggers>
+    <Root level="ERROR">
+      <AppenderRef ref="Sentry"/>
+    </Root>
+  </Loggers>
+</Configuration>
```

Comment:
Is it possible to move this config to `base-java` as well?

Reply:
Thanks for suggestion. Done.